doc/release-notes-1.3.0.rst

```diff
@@ -0,0 +1,571 @@
+  * Added APN readout.
+
+* :ref:`lte_lc_readme` library:
+
+  * Added the concept of events.
+  * Allowed to modify PSM/eDRX parameters at runtime.
+  * Allowed to modify PDP context and PDN authentication at runtime.
+
+* :file:`lib/at_host` library: (?)
+
+  * Updated to use CR termination by default.
+    This reverts the old behavior.
+
+* :ref:`http_application_update_sample` sample:
+
+  * Added support for :ref:`nrfxlib:bsdlib` initialization at system startup.
+  * Changed default download host and filename.
+
+* :ref:`lwm2m_client` sample:
+
+  * Fixed an invalid value of the APN resource in the Connectivity Monitoring object.
+  * Set cell ID information in the Connectivity Monitoring object.
+  * Added delay to the reboot procedure, to allow to send a reply to the server.
+  * Added queue mode support.
+
+* :ref:`asset_tracker` application:
+
+  * Improved stability.
+  * Various bugfixes.
+
+* :ref:`serial_lte_modem` application:
+
+  * Moved from samples to applications.
+  * Added low-power idle mode.
+  * Added support for generic proprietary AT commands.
+  * Added support for BSD socket, TCP/UPD, ICMP, GPS, MQTT, and FTP proprietary AT commands.
+  * Added support for communication to an external MCU over UART.
+  * Added support for transmitting arbitrary hexadecimal data.
+  * Added support for TCP/UDP proxy service (optional).
+  * Added support for SUPL A-GPS.
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated to version 0.9.0.
+    See the :ref:`liblwm2m_carrier_changelog` for detailed information.
+
+* :ref:`nrfxlib:bsdlib`:
+
+  * Updated to version 0.7.3.
+    See the :ref:`nrfxlib:bsdlib_changelog` for detailed information.
+
+* :ref:`supl_client`:
+
+  * Provided version 0.6.0 for download.
+    This new version is required for compatibility with |NCS| v1.3.0.
+
+nRF5
+====
+
+The following changes are relevant for the nRF52 and nRF53 Series. (?)
+
+nRF5340 SoC
+-----------
+
+* Added :ref:`nrf5340_empty_app_core` sample for samples running purely on the nRF5340 network core.
+* When building a Bluetooth sample for nRF5340, the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample is now automatically built as child image.
+
+Multiprotocol Service Layer (MPSL)
+-----------------------------------
+
+See the :ref:`nrfxlib:mpsl_changelog` for detailed information.
+
+* Added TX power envelope functionality.
+* Added support for using a low-swing and full-swing LF clock.
+
+Thread
+------
+
+Added the following samples:
+
+* :ref:`ot_cli_sample` sample - very basic sample with Thread enabled and shell support to allow using the OpenThread Command Line Interface.
+  This sample is needed for Thread certification.
+* :ref:`coap_client_sample` sample and :ref:`coap_server_sample` sample - two samples demonstrating how to use the CoAP protocol over a Thread network.
+* Thread CoAP Multiprotocol Bluetooth LE Client sample - the multiprotocol extension of the :ref:`coap_client_sample` sample.
```

Comment:
There is no multiprotocol support as the driver support was removed.

Reply:
Removed